spec/support/shared_examples/email_header_examples.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'email with inactive white labelling' do |mail|
+  it 'always displays the OFN header with logo' do
+    expect(public_send(mail).body).to include(ContentConfig.url_for(:logo))
+  end
+end
```

Comment:
We could introduce the convention that the mail is always the subject. Even if you define it with `subject(:mail) {...}` then it's still available as `subject`. We can then simplify this code:

```suggestion
RSpec.shared_examples 'email with inactive white labelling' do
  it 'always displays the OFN header with logo' do
    expect(subject.body).to include(ContentConfig.url_for(:logo))
  end
end
```